src/generator-attributes.cs

```diff
@@ -931,19 +925,61 @@ public override string ToString ()
 				builder.AppendLine ("#if __MACOS__");
 				break;
 			case PlatformName.MacCatalyst:
-				builder.AppendLine ("#if __MACCATALYST__");
+				builder.AppendLine ("#if __MACCATALYST__ && !__IOS__ && !__MACOS__");
```

Comment:
The `&& !__MACOS__` part shouldn't be needed, but it doesn't hurt either.

Reply:
Fixed!